api/oss/src/dbs/postgres/triggers/dao.py

```
@@ -279,23 +346,28 @@ async def get_project_and_subscription_by_trigger_id(
+                .limit(2)
             )
 
             result = await session.execute(stmt)
-
-            subscription_dbe = result.scalars().first()
-
-            if not subscription_dbe:
+            subscription_dbes = result.scalars().all()
+            if len(subscription_dbes) != 1:
+                if len(subscription_dbes) > 1:
+                    log.warning(
```

Flake8 (7.3.0)

[error] undefined name 'log'

(F821)